src/EnergyPlus/HeatBalanceKivaManager.cc

```diff
@@ -233,55 +233,88 @@ namespace HeatBalanceKivaManager {
                                                    // about the building at this point)
 
         Real64 Tin;
-        switch (zoneControlType) {
-        case KIVAZONE_UNCONTROLLED: {
-            Tin = assumedFloatingTemp + DataGlobals::KelvinConv;
-            break;
-        }
-        case KIVAZONE_TEMPCONTROL: {
+        if (zoneAssumedTemperature > -999) { // default sets this below -999 at -9999 so uses value if entered
```

Comment:
Can you just use something like `if (zoneAssumedTemperature > Kiva::DefaultWhateverTemp)` rather than a magic number?